Showcase-iOS/DependencyContainer/DependencyContainer.swift

```diff
@@ -93,6 +93,15 @@ struct DependencyContainer {
             return contactUsPresenter
         }
         
+        container.register(AboutPresentable.self) {r in
```

Comment:
This is a valid comment, I know that it will make the code inconsistent but we should call it something else instead of "r" even if its just for the code that you have changed